File: JAVA/PROGRAM/Baju.java
class Baju extends Aksesoris{
    private String untuk;
    private String size;
    private String merk;

    public Baju() {
    }
    public Baju(int id, String nama_produk,  float harga_produk, int stok_produk, String jenis, String bahan, String warna, String untuk, String size, String merk) {
        super(id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna);
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }
    public String getUntuk() {
        return untuk;
    }
    public void setUntuk(String untuk) {
        this.untuk = untuk;
    }
    public String getSize() {
        return size;
    }
    public void setSize(String size) {
        this.size = size;
    }
    public String getMerk() {
        return merk;
    }
    public void setMerk(String merk) {
        this.merk = merk;
    }
}

File: PHP/PROGRAM/Baju.php
<?php 

include 'Aksesoris.php';

class Baju extends Aksesoris {
    private $untuk;
    private $size;
    private $merk;

    public function __construct($untuk = '', $size = '', $merk = '') {
        parent::__construct();
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merk = $merk;
    }

    public function getUntuk() {
        return $this->untuk;
    }

    public function setUntuk($untuk) {
        $this->untuk = $untuk;
    }

    public function getSize() {
        return $this->size;
    }

    public function setSize($size) {
        $this->size = $size;
    }

    public function getMerk() {
        return $this->merk;
    }

    public function setMerk($merk) {
        $this->merk = $merk;
    }


    public function addBaju($id, $nama_produk,  $harga_produk, $stok_produk, $foto, $jenis, $bahan, $warna, $untuk, $size, $merk) {
        parent::setId($id);
        parent::setNama_produk($nama_produk);
        parent::setHarga_produk($harga_produk);
        parent::setstok_produk($stok_produk);
        parent::setFoto_produk($foto);
        parent::setJenis($jenis);
        parent::setBahan($bahan);
        parent::setWarna($warna);
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merk = $merk;
        
    }
}
?>